addons/mail/static/tests/qunit_suite_tests/chatter_tests.js

```diff
@@ -262,6 +262,163 @@ QUnit.test('list activity exception widget with activity', async function (asser
     list.destroy();
 });
 
+QUnit.test('list activity widget: done the acitivty with "ENTER" keyboard shortcut', async function (assert) {
+    assert.expect(5);
+
+    const pyEnv = await startServer();
+    const [mailActivityTypeId1, mailActivityTypeId2] = pyEnv['mail.activity.type'].create([{}, {}]);
+    const [mailActivityId1, mailActivityId2] = pyEnv['mail.activity'].create([
+        {
+            display_name: "Call with Al",
+            date_deadline: moment().format("YYYY-MM-DD"), // now
+            can_write: true,
+            state: "today",
+            user_id: pyEnv.currentUserId,
+            create_uid: pyEnv.currentUserId,
+            activity_type_id: mailActivityTypeId1,
+        },
+        {
+            display_name: "Meet FP",
+            date_deadline: moment().add(1, 'day').format("YYYY-MM-DD"), // tomorrow
+            can_write: true,
+            state: "planned",
+            user_id: pyEnv.currentUserId,
+            create_uid: pyEnv.currentUserId,
+            activity_type_id: mailActivityTypeId2,
+        }
+    ]);
+    pyEnv['res.users'].write([pyEnv.currentUserId], {
+        activity_ids: [mailActivityId1, mailActivityId2],
+        activity_state: 'today',
+        activity_summary: 'Call with Al',
+        activity_type_id: mailActivityTypeId2,
+    });
```

Comment:
Test just requires a single activity/activity subtype, so keep a single one of each.